Formula/zbar.rb

```diff
@@ -21,18 +20,19 @@ class Zbar < Formula
     sha256 x86_64_linux:   "085302f322670cb77040a77645e1972ea14e8e6bc1f6e9ee3e010d53d48dbdd3"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "gettext" => :build
-  depends_on "libtool" => :build
+  head do
+    url "https://github.com/mchehab/zbar.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "gettext" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "pkg-config" => :build
   depends_on "xmlto" => :build
-  depends_on "freetype"
   depends_on "imagemagick"
   depends_on "jpeg-turbo"
-  depends_on "libtool"
-  depends_on "ufraw"
-  depends_on "xz"
```

Comment:
- `freetype` and `libtool` look indirect via `imagemagick`
- `ufraw` doesn't have any usage as far as I could tell. Tried search code base for `ufraw` text but no matches.
- `xz` doesn't seem to be used. Was probably indirect via `imagemagick` a long time ago.